src/main/java/org/eclipse/tractusx/ssi/lib/did/resolver/DidUniResolver.java

```diff
@@ -93,6 +93,7 @@ public DidDocument resolve(Did did) throws DidResolverException {
     } catch (DidResolverException e) {
       throw e;
     } catch (Exception e) {
+      Thread.currentThread().interrupt();
```

Comment:
We should only interrupt the thread when we have `InterruptedException` in the catch.
Here we have a generic `Exception`, can we split the catch block?